### DISCUSSION TITLE
Energy use measured by Utilization or by Provisioned Capacity

### DISCUSSION THREAD

**Discussion by adrianco** [Green-Software-Foundation/sci#328] [SCI feedback]
It's important to distinguish between the carbon used by a marginal increase in power consumption due to a workload, and the total consumption of the system. The system is still there consuming power when it's not being used by a workload, so you could also argue that it's the power usage of the total provisioned capacity of all the systems that's important, not the amount in use by a workload at a point in time. In a cloud environment, you could argue that you only take account of the energy use of the systems you are paying for, but the additional un-billed capacity still uses energy, and your usage ultimately causes the cloud provider to install additional systems.

**Comment by atg-abhishek** [comment]
@adrianco so we're talking here about the idle resource consumption right? Wouldn't that be captured by the TS and RS terms in the [SCI equation](https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/dev/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md#embodied-emissions)? Thoughts on this @Henry-WattTime?

**Reply by Henry-WattTime** [reply]
WG: does provisioned capacity refer to the reserved but unused portion of the hardware? Or the hardware that is not reserved but also not available for other people to use?

HDR: The standard currently accommodates 'reserved' not 'used' resources in the embodied emissions calculation, maybe it needs to be extended to the operational emissions as well.  

Gadhu: This affects both energy and embodied emissions. 
Srini: Even if a system is idle it consumes energy in many ways (monitoring, health, etc.). Applications consume energy even while idle. Managed services are more opaque about what energy would be consumed. Looking for ways to reduce it is important.

**Comment by Henry-WattTime** [comment]
WG: 
Ben: hard to give specific guidance for all solutions. How to treat virtual kit, how to identify boundary
Asim: yes we don't, very challenging issue, maybe good for guidance?
Ben: case study may help illustrate this
Make reference to issue in spec, but not outline hard rules
"The energy consumption should include all hardware reserved or provisioned, not just the hardware actually running"

**Reply by Henry-WattTime** [reply]
Addressed in pull request #345
